src/NodaTime.Web/Markdown/2.0.x/installation.md

```diff
@@ -60,16 +47,3 @@ information on using Noda Time with Json.NET.
 Everything within the NodaTime assembly is in the NodaTime namespace or a "child" namespace. After adding a reference to
 the main assembly (either directly via the file system or with NuGet) and including an appropriate `using` directive, you should
 be able to start using Noda Time immediately, with no further effort.
-
-Debugging
----------
-
-As of version 1.1, the source code of Noda Time is published to [SymbolSource](https://www.symbolsource.org/). You can configure
```

Comment:
We do push to symbol source at the moment, but I'm not at all sure it's useful right now due to the kind of PDB we're building. We need to sort this out in general - will raise an issue when I get the chance...